cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -72,16 +72,6 @@
     {
       "events": [
         {
-          "channel_uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d",
-          "contact_uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3",
-          "created_on": "2000-01-01T00:00:00.000000000-00:00",
-          "id": 0,
-          "step_uuid": "",
```

Comment:
At some point I think we can stop repeating "step_uuid" here as events are also nested under the steps in the path... but whilst we're generating flow step objects in rapidpro, it's a bit awkward to figure out which event occured where without these